### ISSUE TITLE
'Contribute in GitHub' links in blogs

### ISSUE THREAD

**Issue by mbroz2** [OpenLiberty/openliberty.io#1088]

**Comment by NottyCode** [comment]
I don't think we should do this. A blog post is written by an author and I don't think it should suggest shared ownership of the content. Docs are different because they aren't a person's personal output.

**Comment by mbroz2** [comment]
I agree with the con above, but the pro would make it easier for anyone to correct content, grammar, layout, etc issues in the blog (especially true if the blog was created by an external contributor where it might be more difficult to get them to make the change)

**Comment by steven1046** [comment]
I could see the 'edited by' controversial since if someone changes a few words or punctuation then it would say 'edited by person' and if someone else changes it then we would have to include everyone's names even if they didn't change much.

**Comment by steven1046** [comment]
We should be better at catching these grammar errors before publishing instead of having an edit button on the blog. We are working on a PR template to help catch these at review/writing time.

**Comment by kinueng** [comment]
This would be really helpful for the non-English blog posts

**Comment by steven1046** [comment]
Moving this to icebox until we have a compelling reason to implement it.
